src/main/target/SITL/target.h

```diff
@@ -213,22 +226,27 @@ typedef enum
 
 typedef struct {
 	double timestamp;	// in seconds
-	double imu_angular_velocity_rpy[3];	// range: +/- 8192; +/- 2000 deg/se
-	double imu_linear_acceleration_xyz[3];	// sim 1G = 9.81 -> FC 1G = 256
-	double imu_orientation_quat[4];
-	double velocity_xyz[3];
-	double position_xyz[3];
+	double imu_angular_velocity_rpy[3];	// rad/s -> range: +/- 8192; +/- 2000 deg/se
+	double imu_linear_acceleration_xyz[3];	// m/s/s NED, body frame -> sim 1G = 9.80665, FC 1G = 256
```

Comment:
But ArduCopterPlugin define their message as this...
I just reuse the plugin, otherwise we need to write & build plugin ourself.